### ISSUE TITLE
new feature ideas

### ISSUE THREAD

**Issue by Neurofibromin** [kuiperzone/PupNet-Deploy#38]
I think the signing could be made a bit more uniform. Currently there is a separate flag for appimage, flatpak and the InnoSetup sing tool. It would be convenient if I could just specify a gpg file, and all "kinds" sorted out how should they sign using that. Also rpms support signing, but I don't see any way pupnet exposes that functionality. But this is just an idea, not sure how hard would it be to implement (or how useful would it be for others).

manpages support: I would like to provide a manpage for my program. As someone who is not an expert of deb and rpm builds I am not sure how to make the generated manpage file (from markdown, using pandoc) go into the rpm/deb. It would be great if pupnet could solve this - again, not sure how hard this would be to implement.

Thanks for all the work so far, pupnet is a very useful tool!

**Comment by kuiperzone** [comment]
I'd have to look into it in the future. I have a feeling Windows would be the odd one out.

**Comment by kuiperzone** [comment]
Yup. This is good. I will implement in v1.10. Thank you for suggestion.
